### PR TITLE
add obsolete attributes to all v5 classes, methods, and instances

### PR DESCRIPTION
**Pull request by swiftyspiffy**
Relevant issue: https://github.com/TwitchLib/TwitchLib.Api/issues/220

### CODE REVIEW

File: TwitchLib.Api.V5/Users.cs [hunk 1]
```diff
@@ -155,7 +157,7 @@ public async Task<bool> UserFollowsChannelAsync(string userId, string channelId)
         #endregion
 
         #region FollowChannel
-
+        [Obsolete("This is a v5 method, please use a Helix method. All v5 methods will be turned off on February 28, 2022. Details: https://blog.twitch.tv/en/2021/07/15/legacy-twitch-api-v5-shutdown-details-and-timeline/ ")]
```
Comment (iProdigy) [inline TwitchLib.Api.V5/Users.cs:160]:
```suggestion
        [Obsolete("This is a v5 method that was turned off on July 27, 2021. Details: https://discuss.dev.twitch.tv/t/deprecation-of-create-and-delete-follows-api-endpoints/32351 ")]
```

File: TwitchLib.Api.V5/Users.cs [hunk 2]
```diff
@@ -171,7 +173,7 @@ public Task<UserFollow> FollowChannelAsync(string userId, string channelId, bool
         #endregion
 
         #region UnfollowChannel
-
+        [Obsolete("This is a v5 method, please use a Helix method. All v5 methods will be turned off on February 28, 2022. Details: https://blog.twitch.tv/en/2021/07/15/legacy-twitch-api-v5-shutdown-details-and-timeline/ ")]
```
Comment (iProdigy) [inline TwitchLib.Api.V5/Users.cs:176]:
```suggestion
        [Obsolete("This is a v5 method that was turned off on July 27, 2021. Details: https://discuss.dev.twitch.tv/t/deprecation-of-create-and-delete-follows-api-endpoints/32351 ")]
```

**Review by iProdigy — APPROVED** [review]

**Review by Syzuna — CHANGES_REQUESTED** [review]
I would remove the according models as well. like UserFollow or is that still used elsewhere?

**Comment by swiftyspiffy** [comment]
good catch

**Comment by swiftyspiffy** [comment]
Actually, GetUserFollows still uses it, so can't get rid of the model.

**Review by Syzuna — APPROVED** [review]
thought as much. LGTM then

**Pull request merged**